x-pack/plugins/ml/common/util/anomaly_utils.ts

```diff
@@ -42,6 +43,16 @@ export interface EntityField {
   operation?: EntityFieldOperation;
 }
 
+export interface Anomaly {
```

Comment:
Could you adapt the `AnomaliesTableRecord` interface from `ml/common/types/anomalies.ts` rather than exporting a new `Anomaly` interface from here? The new `isCategorizationAnomaly` function is only called from the anomalies tables, so might be possible?

Reply:
Updated here https://github.com/elastic/kibana/pull/120450/commits/7d2a1f0901d0447ef620d8a8ef018ad65c08c5cc